vendor/github.com/containerd/imgcrypt/images/encryption/any.go

```diff
@@ -0,0 +1,42 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package encryption
+
+import "github.com/gogo/protobuf/types"
+
+type anyMap map[string]*types.Any
+
+type any interface {
+	GetTypeUrl() string
+	GetValue() []byte
+}
+
+func fromAny(from any) *types.Any {
```

Comment:
Actually surprised that go allows using `any` here (now that go 1.18 has `any` as alias for `interface{}` https://tip.golang.org/ref/spec#Predeclared_identifiers